### DISCUSSION TITLE
Cleaning up the issues tab

### DISCUSSION THREAD

**Discussion by averybuehler** [shadcn-ui/ui#1216] [General]
The issues tab is slowly becoming a collection of incomplete bug reports, personal suggestions & preferences, and installation errors that are unrelated to the project itself. Along with pull requests, it will likely become unmaintainable soon. Just some food for thought.

**Comment by adeludedperson** [comment]
Exactly, Literally 50% issues are not related to the project (Just suggestions and preferences).
